### PR TITLE
more examples for line protocol

### PR DESCRIPTION
**Pull request by beckettsean**
showing escape characters and proper string quoting

### CODE REVIEW

File: tsdb/README.md [hunk 1]
```diff
@@ -7,13 +7,13 @@ Multiple lines must be separated by the newline character `\n`. The format of th
 [key] [fields] [timestamp]
 ```
 
-Each section is separated by spaces.  The minimum required point consists of a measurement name and at least one field. Points without a specified timestamp will be written using the server's local timestamp.
+Each section is separated by spaces.  The minimum required point consists of a measurement name and at least one field. Points without a specified timestamp will be written using the server's local timestamp. Timestamps are assumed to be in nanoseconds unless a `time_parameter` is passed in the query string.
```
Comment (jwilder) [inline tsdb/README.md]: Should be `precision` when using the HTTP `/write` endpoint.

**Comment by jwilder** [comment]
Small fix but :+1:

File: tsdb/README.md (hunk 1 above)
Comment (beckettsean) [inline tsdb/README.md]: Thanks, updated.

**Comment by beckettsean** [comment]
@jwilder is the query string syntax `precision` or `time_precision`?

**Comment by jwilder** [comment]
For the `/write` endpoint, it's `precision`.

**Comment by jwilder** [comment]
For queries, not sure if there is a parameter.

**Pull request merged**